Makefile

```diff
@@ -63,3 +64,6 @@ lint-errors: images
 
 tests8: images
 	@docker run --rm -v $(shell pwd):/data:Z $(IMAGE)/centos8 pytest
+
+rpms:
```

Comment:
@ZhukovGreen unless it was expected to have the target `rpms` directly, I sugges to call it `build` instead to be consistent with bunch of other tools and projects.

Reply:
So you mean
Make builds? @pirat89

Reply:
It is all three rpms

Reply:
@ZhukovGreen hmm. maybe even `local_build` as it is just for local stuff - it doesn't matter so much whether it's for all systems. when you do the copr build you expect that all expected builds will be created (for every build root unless specified otherwise). I would not do so much difference in this, unless you plan in future provide a target for one particular build. In such a case, maybe `local_build_all` could be what you want. as `build` | `builds` is easy to exchange.

Reply:
But to be honest, I would go most just wiath `local_build` and in case someone would like to specify that only one specific build should be created, I would add possibility to specify an envar for that.

Reply:
I think `make rpms` works fine as well.